geopandas/io/tests/test_file.py

```diff
@@ -1514,6 +1514,40 @@ def test_error_engine_unavailable_fiona(tmp_path, df_points, file_path):
         df_points.to_file(tmp_path / "test.gpkg", engine="fiona")
 
 
+def test_error_monkeypatch_engine_unavailable_pyogrio(
+    monkeypatch, tmp_path, df_points, file_path
+) -> None:
+    # monkeypatch to make pyogrio unimportable
+    monkeypatch.setattr(geopandas.io.file, "_import_pyogrio", lambda: None)
+    monkeypatch.setattr(geopandas.io.file, "pyogrio", None)
```

Comment:
it looks like this line isn't strictly needed, but I suppose ensures consistency.

Reply:
Yeah I think it doesn't hurt and maybe it future proofs the test a bit? I could go either way on it though.